anitya/lib/backends/cran.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+
+"""
+ (c) 2018 - Copyright Elliott Sales de Andrade
+
+ Authors:
+   Elliott Sales de Andrade <quantum.analyst@gmail.com>
+
+"""
+
+
+from anitya.lib.backends import BaseBackend
+from anitya.lib.exceptions import AnityaPluginException
+
+
+def _normalize_version(version):
+    return version.replace('-', '.')
+
+
+class CranBackend(BaseBackend):
+    ''' The custom class for projects hosted on CRAN.
+
+    This backend allows to specify a version_url and a regex that will
+    be used to retrieve the version information.
+    '''
+
+    name = 'CRAN (R)'
+    examples = [
+        'https://cran.r-project.org/web/packages/tidyverse/',
+        'https://cran.r-project.org/web/packages/devtools/',
+    ]
+
+    @classmethod
+    def get_version(cls, project):
+        ''' Method called to retrieve the latest version of the projects
+        provided, project that relies on the backend of this plugin.
+
+        :arg Project project: a :class:`anitya.db.models.Project` object whose backend
```

Comment:
I've been using the ["google style" docbocks](http://www.sphinx-doc.org/en/1.6.3/ext/example_google.html) since I think they look a little nicer in the code. It's not a huge deal since the code base isn't consistent at the moment, but it's what I prefer.